packages/mandelbrot/assets/scss/core/_foundation.scss

```diff
@@ -70,15 +73,22 @@ ul {
 }
 
 a {
-    color: $color-link;
+    --color: rgba(var(--skin-links), 0.9);
+
+    color: rgba($color-link, 0.9);
+    color: var(--color, rgba($color-link, 0.9));
     transition: all 0.15s ease-out;
 
     &:active {
         color: $color-link;
+        color: var(--color, rgba($color-link, 0.9));
     }
 
     &:hover {
-        color: darken($color-link, 15%);
```

Comment:
i changed the logic here because if we want to use darken with css variables, i'd need to use hsl everywhere and that would be even more complicated code-wise. using rgba everywhere is just simpler.